docs/source/basic_tutorials/notebook.md

```diff
@@ -401,6 +401,25 @@ args = ("fp16", 42, 64)
 notebook_launcher(training_loop, args, num_processes=2)
 ```
 
+In the case of running on multiple nodes, you need to set up a jupyter session at each node and run the launching cell at the same time,
+or use [papermill](https://papermill.readthedocs.io/en/latest/) to execute the notebook.
+
+For an environment containing 2 nodes with 4 GPUs per node, the master launcher of the first node with address "172.31.43.8" would look like so:
```

Comment:
```suggestion
For an environment containing 2 nodes (computers) with 4 GPUs each and the main computer with an IP address of "172.31.43.8", it would look like so:
```